presto-main/src/main/java/com/facebook/presto/metadata/MetadataManager.java

```diff
@@ -601,16 +601,16 @@ public TableStatisticsMetadata getStatisticsCollectionMetadata(Session session,
     }
 
     @Override
-    public AnalyzeTableMetadata getAnalyzeTableMetadata(Session session, TableHandle tableHandle, Map<String, Object> properties)
+    public AnalyzeTableMetadata getAnalyzeTableMetadata(Session session, TableHandle tableHandle, TableMetadata tableMetadata, Map<String, Object> properties)
     {
         ConnectorId connectorId = tableHandle.getConnectorId();
         CatalogMetadata catalogMetadata = getCatalogMetadataForWrite(session, connectorId);
         ConnectorMetadata metadata = catalogMetadata.getMetadata();
 
-        ConnectorAnalyzeTableMetadata analyzeTableMetadata = metadata.getAnalyzeTableMetadata(session.toConnectorSession(connectorId), tableHandle.getConnectorHandle(), properties);
+        ConnectorAnalyzeTableMetadata analyzeTableMetadata = metadata.getAnalyzeTableMetadata(
+                session.toConnectorSession(connectorId), tableHandle.getConnectorHandle(), tableMetadata.getMetadata(), properties);
```

Comment:
style: all args to `metadata.getAnalyzeTableMetadata` on single line or each on a separate line